src/main/java/org/mastodon/mamut/tomancak/lineage_registration/RefMapUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.mastodon.mamut.tomancak.lineage_registration;
+
+import java.util.function.BiConsumer;
+
+import org.mastodon.collection.ObjectRefMap;
+
+public class RefMapUtils
+{
+	// TODO: This needs to become the default implementation of ObjectRefMap.forEach().
```

Comment:
Can you make an issue or resolve this TODO?

Reply:
Done https://github.com/mastodon-sc/mastodon-collection/pull/13

Reply:
The todo is still in source code.

Reply:
I updated the TODO accordingly

Reply:
ok